docs/team-process.md

```diff
@@ -59,20 +59,34 @@ Every week we rotate responsibility for updating dependencies. As an open-source
 > After changes are made to any dependencies, deploy to an exp environment to verify that all tests pass!
   > * If terraform needs to be updated, deploy locally to an exp environment first. 
 
-#### JavaScript Library Updates
+#### Library Update Steps
 
-`npm update`: Update to current minor versions of all libraries. These shouldn't include any breaking changes, but still might, so it's best to verify with smoke tests in AWS.
+1. `npm update`: Update to current minor versions of all libraries. These shouldn't include any breaking changes, but still might, so it's best to verify with smoke tests in AWS.
 
-`npm outdated`: Informs us of major version updates that we need to update manually. Often there are breaking API changes that require refactoring.
+2. `npm outdated`: Informs us of major version updates that we need to update manually. Often there are breaking API changes that require refactoring.
 
-`npm audit`: Informs us of known security vulnerabilities. If transitive dependencies are vulnerable, use the resolutions block in `package.json` to specify version overrides. 
+3. `npm audit`: Informs us of known security vulnerabilities. If transitive dependencies are vulnerable, use the resolutions block in `package.json` to specify version overrides. 
 If dependencies have no patch, replace it with an alternative, or wait for the library to be patched.
 
-#### Infrastructure as Code Libraries
+    NOTE: If any npm packages are updated, update the node cache version in the circle config. 
```

Comment:
How does one do this?

Reply:
https://github.com/flexion/ef-cms/commit/959025d26dd28977253f62ad4f0fa2a39706ed03 is an example, but in addition to these 2 locations you'd replace all instances of v9 within `config.yml`. I can add that specifically to these docs